pkgs/development/libraries/fontconfig/default.nix

```diff
@@ -57,7 +58,7 @@ stdenv.mkDerivation rec {
 
   postInstall = ''
     cd "$out/etc/fonts"
-    "${libxslt.bin}/bin/xsltproc" --stringparam fontDirectories "${dejavu_fonts.minimal}" \
+    "${if (stdenv.hostPlatform != stdenv.buildPlatform) then buildPackages.libxslt.bin else libxslt.bin}/bin/xsltproc" --stringparam fontDirectories "${dejavu_fonts.minimal}" \
```

Comment:
Should `buildPackages.libxslt.bin` not always return the correct binary?
cc @matthewbauer

Reply:
buildPackages work fine when not cross compiling. It's just that ```pkgs = buildPackages = buildPackages.buildPackages``` in this case.

I would recommend putting libxslt.bin in nativeBuildInputs to avoid this logic though.